jsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this a new file??
seems like its being used for linting, but why its added now, it should have been there already.
If not, dont add this